src/NHibernate.Test/CacheTest/JsonSerializerCacheFixture.cs

```diff
@@ -0,0 +1,525 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using NHibernate.Caches.CoreDistributedCache;
+using NHibernate.Caches.CoreDistributedCache.Memory;
+using NHibernate.Caches.Util.JsonSerializer;
+using NHibernate.Cfg;
+using NHibernate.Linq;
+using NHibernate.Multi;
+using NHibernate.Transform;
+using NUnit.Framework;
+using Environment = NHibernate.Cfg.Environment;
+
+namespace NHibernate.Test.CacheTest
+{
+	[TestFixture]
+	public class JsonSerializerCacheFixture : TestCase
```

Comment:
Why it's implemented as separate new class? Can't it be based on existing one (NativeSQLQueriesFixture?) with configuration for using JsonSerializer cache

Reply:
It is a mix of NativeSQLQueriesFixture and BatchableCacheFixture, for checking non-sql-native queries too.